railties/lib/rails/rack/logger.rb

```diff
@@ -7,6 +7,9 @@
 module Rails
   module Rack
     # Sets log tags, logs the request, calls the app, and flushes the logs.
+    #
+    # Log tags can be an Array containing: methods that the <tt>request</tt> object responds to, a Proc
```

Comment:
@sshaw yea that's true but since these docs are close to the source it's still nice to know where the "Log tags" are stored. What about: `Log tags (+taggers+) can be ...`